reverse_proxy.go

```diff
@@ -97,6 +97,10 @@ var hopHeaders = []string{
 // NewSingleHostReverseProxy does not rewrite the Host header.
 // To rewrite Host headers, use ReverseProxy directly with a custom
 // director policy.
+//
+// Note: it will use the default global client.Client instance when you do not pass any config.ClientOption.
```

Comment:
when you do not pass any config.ClientOption. -> if no config.ClientOption is passed.

Reply:
done